dev/custom-pages/space.scss

```diff
@@ -0,0 +1,10 @@
+.ucf-section-space-ignite-callout .ignite-logo {
+  background: url('//www.ucf.edu/wp-content/uploads/2017/01/ignite-logo.png') no-repeat;
```

Comment:
If we move the ignite-logo class into the theme, let's copy this logo file into the theme as well so that it doesn't break if the image is ever removed from the media library.